service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,155 @@
+            <set field="statusCodes" from="resolvedAddresses?resolvedAddresses.attributes:null"/>
+            <if condition="responseMap==null || responseMap == '' || statusCodes.InterpolatedStreetAddress=='true'
+                || statusCodes.DPV=='false' || statusCodes.AddressType!='STANDARDIZED' || statusCodes.Resolved=='false'">
+                <then><set field="trustLevelEnumId" value="CmtlInvalid"/></then>
+                <else><set field="trustLevelEnumId" value="CmtlValid"/></else>
+            </if>
+            <!-- Updating trustLevelEnumId value -->
+            <log message="Fedex validate address ${contactMechId} trust ${trustLevelEnumId}"/>
+            <entity-find-one entity-name="mantle.party.contact.ContactMech" value-field="contactMech"/>
+            <set field="contactMech.trustLevelEnumId" from="trustLevelEnumId"/>
+            <entity-update value-field="contactMech"/>
+
+            <!-- Error Handling -->
+            <if condition="resolvedAddresses.customerMessages &amp;&amp; resolvedAddresses.customerMessages[0]">
+                <iterate list="resolvedAddresses.customerMessages" entry="alert">
+                    <message type="danger">Fedex error ${alert.code}: ${alert.message}</message>
+                </iterate>
+            </if>
+            <if condition="responseMap.output.alerts &amp;&amp; responseMap.output.alerts[0]">
+                <iterate list="responseMap.output.alerts" entry="alert">
+                    <message type="danger">Fedex error(${alert.alertType}) - ${alert.code}: ${alert.message}</message>
+                </iterate>
+            </if>
+        </actions>
+    </service>
+
+    <!-- Returns RequestMap for validate#PostalAddress service -->
+    <service verb="get" noun="AddressMap">
+        <in-parameters>
+            <parameter name="postalContactMechId" required="true"/>
+            <parameter name="telecomContactMechId"/>
+            <parameter name="partyId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="addressMap" type="Map"/>
+        </out-parameters>
+        <actions>
+
+            <!-- Searching PostalAddress -->
+            <entity-find-one entity-name="mantle.party.contact.PostalAddress" value-field="postalAddress">
+                <field-map field-name="contactMechId" from="postalContactMechId"/>
+            </entity-find-one>
+            <if condition="!postalAddress">
+                <return type="danger" message="In get#AddressMap service, postal address not found."/>
+            </if>
+
+            <!-- Searching State/Province Code -->
+            <entity-find entity-name="Geo" list="geoStateProvinceData">
+                <econdition field-name="geoId" from="postalAddress.stateProvinceGeoId"/>
+                <econditions combine="or">
+                    <econdition field-name="geoTypeEnumId" value="GEOT_STATE"/>
+                    <econdition field-name="geoTypeEnumId" value="GEOT_PROVINCE"/>
```

Comment:
I think you can also use operator="in" as the condition is on the multiple values for the same field.
eg. <econdition field-name="geoTypeEnumId" operator="in" value="GEOT_STATE,GEOT_PROVINCE"/>